packages/opencensus-exporter-object/package.json

```diff
@@ -0,0 +1,66 @@
+{
+	"name": "@opencensus/exporter-object",
+	"version": "0.0.13",
+	"description": "OpenCensus Object Exporter allows the user to collect and access traces with OpenCensus Node.js.",
```

Comment:
Do you want to publish this package to npm? If no, can we add `private: true`?

Reply:
Because looks like real use case is for unit testing only, right?

Reply:
How could a developer consume it without npm? I'd expect it will be installed as a dev dependency.

Reply:
Makes sense. I thought, intention was to use in this project only.

Reply:
No, sorry for not explaining the context well. I meant testing instrumentation in external services, My use case is that I instrumented an application with OpenCensus and now I want to be sure it collects the right spans during the application lifecycle. I could stub on Tracer internals but I feel it would be a cleaner approach.